tasks/testnet.js

```diff
@@ -1,27 +1,53 @@
 "use strict"
 
-let runDev = require(`./runner.js`)
+let runner = require(`./runner.js`)
 let config = require(`../app/src/config.js`)
-const path = require(`path`)
-const userHome = require(`user-home`)
-const { startLocalNode } = require(`./gaia.js`)
+const fs = require(`fs-extra`)
+const { join, resolve } = require(`path`)
+const {
+  startNodes,
+  buildNodes,
+  setupAccounts
+} = require(`./build/local/helper`)
+const appDir = resolve(`${__dirname}/../`)
+const buildTestnetPath = join(appDir, `builds`, `testnets`)
 
 async function main() {
   const network = process.argv[2] || config.default_network
+  const numberNodes = parseInt(process.argv[3], 10) || 1
 
   let extendedEnv = {}
+  let networkPath = `./app/networks/${network}/`
 
   if (network === `local-testnet`) {
+    const { cliHomePrefix, nodes, mainAccountSignInfo } = await buildNodes(
+      join(buildTestnetPath, network),
+      {
+        chainId: network,
+        password: `1234567890`,
+        overwrite: false,
+        moniker: `local`,
+        keyName: `main-account`
+      },
+      numberNodes
+    )
+    await startNodes(nodes, mainAccountSignInfo, network)
+    fs.copySync(join(nodes[1].home, `config`), cliHomePrefix)
```

Comment:
This mimic the behaviour of electron start, I tried hard not to put those three lines here and rely on that hook but I couldn't find a nice way of doing it. Any idea?

Reply:
From copySync to WriteFileSync should happen at boot in my mind, instead I'm forcing it to happen at network creation time (I didn't wanted to touch also that process right now, to keep this PR small)

Reply:
boot process will die anyway ;)